R/rating.R

```diff
@@ -1,22 +1,45 @@
-fomantic_size_levels <- c("mini", "tiny", "small", "", "large", "huge", "massive")
+#' allowed sizes
+FOMANTIC_SIZE_LEVELS <- c("mini", "tiny", "small", "", "large", "huge", "massive")
 
+#' Rating
+#'
+#' Crates rating component
+#'
+#' @param name The \code{input} slot that will be used to access the value.
+#' @param label the contents of the item to display
+#' @param value initial rating value
+#' @param max maximum value
+#' @param icon An optional \code{\link{uiicon}()} to appear on the button.
+#' @param color character with colour name
+#' @param size character with legal semantic size, eg. "medium", "huge", "tiny"
+#'
 #' @export
-rating <- function(inputId, label, value = 0, max = 3, icon = "star", color = "yellow", size = 4) {
-
-  class <- glue::glue("ui {fomantic_size_levels[size]} {color} rating")
+rating <- function(name, label = "", value = 0, max = 3, icon = "star",
+                   color = "yellow", size = "") {
```

Comment:
We need to rethink the way sizes are used in shiny.semantic. For me using integers is much easier as I don't need to remember size codes. I think from user perspective this thing can look similar. Wdyt?

Reply:
Okay I've seen you comment. Let's leave it as is.

Reply:
Here I'm not sure, my main argument is that numbers might be easily confused with font-size and cause frustration why eg. 14 doesn't work. But this is a UX question, let's leave it aside for now. Other objects (eg. actionbutton) use the text formatting now, so let's keep it consistent, but I'll open an issue to address that and normalise among all functions if necessary (#186)